railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -3,9 +3,6 @@
   <head>
     <title><%= camelized %></title>
     <%%= csrf_meta_tags %>
-    <%- unless options[:skip_action_cable] -%>
-    <%%= action_cable_meta_tag %>
```

Comment:
+1 let's remove the method, and just document how to build the meta tag "manually" if you need to specify a custom mount point.